lib/Doctrine/Expression/Driver.php

```diff
@@ -302,12 +301,10 @@ public function substring($value, $from, $len = null)
      * concat() accepts an arbitrary number of parameters. Each parameter
      * must contain an expression or an array with expressions.
      *
-     * @param string|array(string) strings that will be concatinated.
+     * @param string ...$args strings that will be concatinated.
      */
-    public function concat()
+    public function concat(...$args)
```

Comment:
Changing to use variadic (available since php 5.6, which is minimum supported version).